app/helpers/events_helper.rb

```diff
@@ -0,0 +1,5 @@
+module EventsHelper
+  def attending(user, event)
+    user.attendances.where(event_id: event.id).empty?
```

Comment:
**OPTIONAL**
Event if it's working, what if, in the `Attendances` model you have a method or a scope for that.
I think the code will be soft and more pro